contracts/starknet/assertions.cairo

```diff
@@ -0,0 +1,123 @@
+from starkware.cairo.common.bitwise import bitwise_and
+from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin
+from starkware.cairo.common.math import assert_not_zero
+from starkware.cairo.common.uint256 import (
+    Uint256,
+    uint256_eq,
+    uint256_lt,
+    uint256_signed_le,
+    uint256_signed_lt,
+    uint256_le,
+    uint256_check,
+)
+from contracts.starknet.safe_math import Int256
+
+func either(a: felt, b: felt) -> (res: felt) {
+    if (a + b == 0) {
+        return (0,);
+    }
+    return (1,);
+}
+
+func assert_either(a: felt, b: felt) {
+    if (a + b == 0) {
+        assert 1 = 0;
+    }
+    return ();
+}
+
+func both(a: felt, b: felt) -> (res: felt) {
+    if (a + b == 2) {
+        return (1,);
+    }
+    return (0,);
+}
+
+func assert_both(a: felt, b: felt) {
+    assert a + b = 2;
+    return ();
+}
+
+func not_0(a: Uint256) -> (res: felt) {
+    if (a.low + a.high == 0) {
+        return (0,);
+    }
+    return (1,);
+}
+
+func assert_not_0(a: Uint256) {
+    assert_not_zero(a.low + a.high);
+    return ();
+}
+
+func assert_0(a: Uint256) {
+    assert a.low + a.high = 0;
+    return ();
+}
+
+func ge{range_check_ptr}(a: Uint256, b: Uint256) -> (res: felt) {
+    let (lt: felt) = uint256_lt(a, b);
+    return (res=1 - lt);
+}
+
+// signed!
+func _ge_0{range_check_ptr}(a: Uint256) -> (res: felt) {
+    let (res) = uint256_signed_le(Uint256(low=0, high=0), a);
+    return (res,);
+}
+
+func le{range_check_ptr}(a: Uint256, b: Uint256) -> (res: felt) {
+    alloc_locals;
```

Comment:
There is an `uint256_le` in the std lib now.

Reply:
My intention was to use `uint256_le` in implementation of `le` not to replace `le` with `uint256_le`.